Sources/View/ReusableView.swift

```diff
@@ -20,15 +20,19 @@ public extension ReusableView where Self: UITableViewCell {
     init() {
         self.init(style: .default, reuseIdentifier: Self.reuseIdentifier)
 
-        setupLayout()
+        setupSubviews()
```

Comment:
I have mixed feelings about this. Even though grammatically `setUp` is probably more correct, visually I think I prefer `setup` 😆 

@portellaa, @mschuetz72, @jakewelton opinions?